### ISSUE TITLE
objects don't show up for some time

### ISSUE THREAD

**Issue by natrius** [OpenBeerMap/OpenBeerMap.github.io#73]
https://openbeermap.github.io/#17/47.80493/13.04823
there should be some, at least this one https://www.openstreetmap.org/node/470891768

Actually, after some scrolling and zooming around some cafes seem to show on the west of Salzburg. After F5 some time, after CTRL-F5 some times, zooming in and out and in again they start to show up. 

Thats annoysing and it seems others had the same problem. https://github.com/OpenBeerMap/OpenBeerMap.github.io/issues/72 Maybe the caching need some time there and when its at the server it shows faster for everyone?

**Comment by nlehuby** [comment]
Hey,

thanks for your feedback @natrius. 
There are some bars here, including the one your are pointing at:
![image](https://user-images.githubusercontent.com/919962/74017133-efc66c00-4993-11ea-994b-61ae7efe6427.png)


As I can reproduce it, I suspect it is probably an issue with the overpass API quota limits :thinking:

**Comment by natrius** [comment]
Yes, i asked some friends and they had the same problem. It shows up but just after some time. :/

**Comment by natrius** [comment]
Same problem with Firefox, Chrome and Edge, btw. Working with Windows10 here, but also tried it with KDE neon (Ubuntu).
